system_test/aest_sync_SUITE.erl

```diff
@@ -705,14 +707,48 @@ net_split_mining_power(Cfg) ->
 
     ok.
 
-quick_start_stop(Cfg) ->
-    setup_nodes(cluster([n1, n2], #{}), Cfg),
+abrupt_stop_new_node(_Cfg) ->
+    {skip, database_restart_needs_fix};
+abrupt_stop_new_node(Cfg) ->
+    RepairTimeout = 30000, % Time allowed for node to repair DB and finish sync
+    Nodes = [n1, n2],
+    setup_nodes(cluster(Nodes, #{}), Cfg),
+    % Start both nodes
+    [start_node(N, Cfg) || N <- Nodes],
+    % Stop node 2 abruptly
+    stop_node(n2, 2000, Cfg),
+    % Restart node 2
+    start_node(n2, Cfg),
+    % Check that they synchronize
+    Blocks = wait_for_value({height, 5}, Nodes, RepairTimeout, Cfg),
+    assert_in_sync(Blocks).
+
+abrupt_stop_mining_node(_Cfg) ->
+    {skip, database_restart_needs_fix};
+abrupt_stop_mining_node(Cfg) ->
+    RepairTimeout = 30000, % Time allowed for node to repair DB and finish sync
+    ShutdownTimeout = proplists:get_value(node_shutdown_time, Cfg),
+    Nodes = [n1, n2],
+    setup_nodes(cluster(Nodes, #{}), Cfg),
+    % Start both nodes
+    [start_node(N, Cfg) || N <- Nodes],
+    % Let them build some chain
+    Blocks = wait_for_value({height, 5}, Nodes, 5 * ?MINING_TIMEOUT, Cfg),
+    assert_in_sync(Blocks),
+    % Stop node 2
+    stop_node(n2, ShutdownTimeout, Cfg),
```

Comment:
Re usage of stop_node in this test case - same as above. To be clarified how abruptly node shall be stopped.